tests/tests.py

```diff
@@ -67,6 +66,83 @@ def test_piranha_match_only():
             assert rule 
             assert _is_readable(str(match))
 
+
+def test_insert_field_add_import():
+    add_field_declaration = Rule (
+        name= "add_field_declaration",
+        query= 
+        """(
+            (class_declaration name: (_)@class_name 
+                body : (class_body ((_)*) @class_members)  @class_body
+            ) @class_declaration
+            (#eq? @class_name "FooBar")
+        )
+        """,
+        replace_node="class_body",
+        replace="""{
+  private List<String> names;\n @class_members
+}""",
+        constraints= set([
+            Constraint(
+                matcher= "(class_declaration ) @c_cd",
+                queries = ["""
+                (
+                    (field_declaration (variable_declarator name:(_) @name )) @field
+                    (#eq? @name "names")
+                )"""]
+            )
+        ]),
+    )
+
+    add_import = Rule (
+        name= "add_import",
+        query= "(package_declaration) @pkg_dcl",
+        replace_node="pkg_dcl",
+        replace="""@pkg_dcl
+import java.util.List;
+""",
+        groups= set(["Cleanup Rule"]),
+        constraints= set([
+            Constraint(
+                matcher= "(program ) @prgrm",
+                queries = ["""
+                (
+                    (import_declaration (scoped_identifier) @type ) @import
+                    (#eq? @type "java.util.List")
+                )"""]
+            )
+        ]),
+    )
+
+    edge1 = OutgoingEdges(
+            "add_field_declaration",
+            ["add_import"],
+            "File"
+        )
+
+    rule_graph = RuleGraph(
+        rules= [add_field_declaration, add_import],
+        edges = [edge1]
+        )
+
+    args = PiranhaArguments(
+        "test-resources/java/insert_field_and_import/input",
+        "test-resources/java/insert_field_and_initializer/configurations",
+        "java",
+        {
+            "stale_flag_name": "STALE_FLAG",
+            "treated": "true",
+            "treated_complement": "false",
```

Comment:
Why do we need any sort of configuration about stale flags to run the field insertion example above? Seems unrelated to flags.